exercises/practice/robot-simulator/.meta/config.json

```
@@ -1,6 +1,13 @@
 {
   "blurb": "Write a robot simulator.",
-  "authors": [],
+  "authors": [
+    "Corentin-Leffy"
+  ],
+  "contributors": [
+    "devkabiir",
+    "ErikSchierboom",
+    "Stargator"
+  ],
```

```suggestion
  "contributors": [
  ],
```

---

I suggest adding my name due to refactoring for NNBD to maintain current functionality.

```suggestion
  "contributors": [
    "Stargator"
  ],
```